dashboard/config/locales/scripts.en.yml

```diff
@@ -167,6 +167,7 @@ en:
           lessons:
             Play Lab:
               name: Play Lab
+          description_audience: ''
```

Comment:
All of the changes to scripts.en.yml came from updating old scripts that to fix weird diff when I checked eveything by hand

Reply:
Ya sorry that sentence did not make sense. Yes I went in to the edit page for the script and saved it.